po/libbytesize.pot

```diff
@@ -0,0 +1,103 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Red Hat, Inc.
+# This file is distributed under the same license as the libbytesize package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: libbytesize 2.1\n"
+"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
```

Comment:
Might as well update/fix this as part of the changes here.

Reply:
Fixed.